backend/python/app/models/comment.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy import inspect
+from sqlalchemy.orm.properties import ColumnProperty
+
+from . import db
+
+
+class Comment(db.Model):
+
+    __tablename__ = "comments"
+
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    story_translation_id = db.Column(
+        db.Integer, db.ForeignKey("story_translations.id"), index=True, nullable=False
```

Comment:
woops my bad for the typo in the ticket - what you want is a foreign key to `story_translation_contents`, not `story_translations` (there are many `comments` for one `story_translation_contents`)